ui-lib/build.gradle.kts

```diff
@@ -120,6 +122,7 @@ dependencies {
     implementation(projects.sdkExtLib)
     implementation(projects.spackleAndroidLib)
     api(projects.uiDesignLib)
+    implementation("androidx.graphics:graphics-shapes-android:1.0.0-rc01")
```

Comment:
Is this new dependency necessary? If so, please move it into the `gradle.properties` and `settings.gradle` files.

Reply:
it can be removed :)